dhcpv4/ztpv4/ztp.go

```diff
@@ -71,6 +95,32 @@ func ParseVendorData(packet *dhcpv4.DHCPv4) (*VendorData, error) {
 		return vd, nil
 	}
 
-	// We didn't match anything.
-	return nil, errors.New("no known ZTP vendor found")
+	return vd, nil
+}
+
+// ParseVendorData will try to parse dhcp4 options looking for more
+// specific vendor data (like model, serial number, etc).
+func ParseVendorData(packet *dhcpv4.DHCPv4) (*VendorData, error) {
+	vd, err := parseVIVC(packet)
```

Comment:
NIT: can you call first the function that parses the lower option number?

Reply:
Called parseClassIdentifier before parseVIVC